src/backend/src/services/teams.services.ts

```diff
@@ -30,6 +30,19 @@ export default class TeamsService {
     return teams.map(teamTransformer);
   }
 
+  /**
+   * Gets all teams
```

Comment:
This needs to be updated

Reply:
Could you also please add a little part to the jsdoc of `getAllTeams` saying archived teams arent included